osquery/utils/system/windows/env.cpp

```diff
@@ -54,4 +59,51 @@ boost::optional<std::string> getEnvVar(const std::string& name) {
   return std::string(buf.data(), value_len);
 }
 
+boost::optional<std::string> expandEnvString(const std::string& input) {
+  std::vector<char> buf;
+  buf.assign(kInitialBufferSize, '\0');
+
+  // ExpandEnvironmentStrings doesn't support inputs larger than 32k.
+  if (input.size() > 32768) {
+    return boost::none;
```

Comment:
Is it worth reporting a warning/verbose message here? in the event something isn't expanded because it was too large, might be nice to be notified of it?